From 375ed9ae99b66ecb017a8a24c71c612a94ee853d Mon Sep 17 00:00:00 2001
From: freedesktop_sdk_updater <freedesktop_sdk_updater@libreml.com>
Date: Sun, 30 Nov 2025 00:53:29 +0000
Subject: [PATCH 1/2] Update elements/components/libinput.bst to 1.30.0-0

Updates elements/components/libinput.bst to 1.30.0-0 (11b97ed)
Previous version was 1.29.2-0 (9177831)
---
 elements/components/libinput.bst | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/elements/components/libinput.bst b/elements/components/libinput.bst
index 1598cf03bd9..b74780aec68 100644
--- a/elements/components/libinput.bst
+++ b/elements/components/libinput.bst
@@ -25,4 +25,4 @@ sources:
   track: '*.*.*'
   exclude:
   - "*.9??"
-  ref: 1.29.2-0-g917783150a5bab7d55eedc5714f1b49b192426ac
+  ref: 1.30.0-0-g11b97edcdc369177f8567d79614ea699281f813a
-- 
GitLab


From 7f3ba2786567ca9db9df199028e65468e56beb3e Mon Sep 17 00:00:00 2001
From: bbhtt <bbhtt.zn0i8@slmail.me>
Date: Sun, 30 Nov 2025 14:33:08 +0530
Subject: [PATCH 2/2] libinput: Disable lua plugins

---
 elements/components/libinput.bst | 1 +
 1 file changed, 1 insertion(+)

diff --git a/elements/components/libinput.bst b/elements/components/libinput.bst
index b74780aec68..245a3155d94 100644
--- a/elements/components/libinput.bst
+++ b/elements/components/libinput.bst
@@ -18,6 +18,7 @@ variables:
     -Ddocumentation=false
     -Dtests=false
     -Dzshcompletiondir=no
+    -Dlua-plugins=disabled
 
 sources:
 - kind: git_repo
-- 
GitLab

